cludo/utils.go

```diff
@@ -27,7 +28,29 @@ func GenerateEnvironment(cc *config.ClientConfig, target string, keys []string,
 	if err != nil {
 		return nil, err
 	}
-	signer, err := cc.NewDefaultSigner(keys)
+
+	var response *environment.GenerateEnvironmentOK
+
+	for _, key := range keys {
```

Comment:
I'm not a gopher, but it doesn't look to me like we have a test showing that this loop is working as expected.

Reply:
So, I agree, that we need a test here (and actually for ALL the CLI entrypoints), but I am not sure how to go about this.

It feels like we need to implement testing of the cobra/viper components (basically all the CLI options) and create some integration testing for logic that really requires the client and server to talk to each other for useful testing to occur.

I'll create to tasks for these, as they both feel like a decent chunk of work.